vent/coordinator/ipc.py

```diff
@@ -0,0 +1,28 @@
+import threading
```

Comment:
Don't know if you already have a plan here but really highly recommend zeromq

http://zguide.zeromq.org/py:all
https://pyzmq.readthedocs.io/en/latest/

Reply:
My preference is to avoid an intermediary messaging process. Primarily because it's one more process that has to be managed and that can fail. We can discuss more, but I just want to keep it as simple with fewest components.